nva-datamodel-java/src/main/java/no/unit/nva/model/EntityDescription.java

```diff
@@ -101,18 +103,39 @@ public List<Contributor> getContributors() {
 
     public void setContributors(List<Contributor> contributors) {
         this.contributors = Objects.nonNull(contributors)
-                                ? contributors.stream().sorted(this::compareContributors).collect(Collectors.toList())
-                                : contributors;
+                ? extractContributors(contributors)
+                : contributors;
+    }
+
+    private List<Contributor> extractContributors(List<Contributor> contributors) {
+        var contributorList = contributors.stream()
+                .sorted(Comparator.comparing(Contributor::getSequence, Comparator.nullsLast(Comparator.naturalOrder())))
+                .collect(Collectors.toList());
```

Comment:
fixed! this resulted in me refactoring updatedContributorSequence into a immutable method also (because toList is immutable and collect(Collectors.toList()) is not)